docs/docs/marketplace/marketplace_overview.md

```diff
@@ -0,0 +1,95 @@
+---
+id: marketplace-overview
+title: 'Marketplace : Overview'
+---
+
+# Marketplace : Overview
+
+With ToolJet Marketplace, ToolJet users can conveniently add custom plugins (datasources) to their workspaces. This feature enables users to create plugins that cater to their specific needs and integrate them seamlessly with ToolJet.
+
+<div style={{textAlign: 'center'}}>
+
+<img className="screenshot-full" src="/img/marketplace/overview/marketplace.png" alt="Marketplace" />
+
+</div>
+
+## Enabling the marketplace 
+
+To **Activate** the marketplace feature, users need to add the following environment variable to their **[`.env`](/docs/setup/env-vars#marketplace)** file:
+
+```bash
+ENABLE_MARKETPLACE_FEATURE=true
+```
+
+Once the marketplace feature has been activated, a Marketplace icon will appear on the left-hand sidebar of the dashboard, providing users with access to the Marketplace.
```

Comment:
An additional step needs to be included in the instructions to ensure that all plugins are available when running locally. Specifically, building the marketplace first and then starting the server is necessary.